agent/charms/testflinger-agent-host-charm/src/charm.py

```diff
@@ -81,13 +82,24 @@ def copy_ssh_keys(self):
             self._stored.ssh_pub = pub_key
             self.write_file("/home/ubuntu/.ssh/id_rsa.pub", pub_key)
 
-    def copy_tf_cmd_scripts(self):
+    def update_tf_cmd_scripts(self):
+        """Update tf_cmd_scfipts"""
         tf_cmd_dir = "src/tf-cmd-scripts/"
+        usr_local_bin = "/usr/local/bin/"
+        for file in os.listdir(usr_local_bin):
+            if "tf" in file:
+                os.remove(os.path.join(usr_local_bin, file))
```

Comment:
I don't think there's a need to remove these. shutil.copy() will overwrite them. Also, although it's unlikely, you could accidentally remove some unrelated binary that happens to have "tf" in the filename.